index.js

```diff
@@ -149,6 +195,63 @@ app.delete('/events/:id', async (req, res) => {
     res.status(405).json(jsonHelper(405, "Method not allowed"))
 })
 
+// /participant/:eventId route
+
+app.get('/participant/:eventId', async (req, res) => {
+    res.status(405).json(jsonHelper(405, 'Method not allowed'))
+})
+
+app.post('/participant/:eventId', dbMiddleware, async (req, res) => {
+    const _id = ObjectId(req.params.eventId)
+    const id = ObjectId()
```

Comment:
Suggest making these names more semantic - something like `eventId` and `newParticipantId`